sys-libs/pam/pam-1.7.0_p20241230-r4.ebuild

```diff
@@ -105,6 +105,7 @@ multilib_src_configure() {
 		$(meson_feature nis)
 		$(meson_feature nls i18n)
 		$(meson_feature selinux)
+		$(meson_feature docs)
```

Comment:
This is trivially verifiably wrong. You took an argument that was formerly passed only when building for the native ABI in a multilib unified build. You replaced it by passing the option whenever docs is enabled, for both ABIs.